source/switching_to_live/index.html.md.erb

```diff
@@ -34,6 +34,8 @@ During the go live process, you'll need to:
 
 Select your test ('sandbox') account or your test Stripe account in the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/my-services), then select __Request a live account__.
 
+If you have a Worldpay test account, do not request a live account from it. Use your sandbox test account instead.
```

Comment:
I'm trying something out so it follows on from the lead in line a bit better. Is it still accurate this way round?
```suggestion
Use the sandbox for testing your Worldpay test account. Do not request a live account from it. 
```

Reply:
This way around wouldn't make sense any more, but I do think it could be better. Maybe:

```suggestion
Do not request a live account from a Worldpay test account. Always use your sandbox account to go live.
```

Reply:
sounds good